README.md

```diff
@@ -1,23 +1,29 @@
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A523.04.3-brightgreen.svg)](https://www.nextflow.io/)
 
-# Example Pipeline for IRIDA Next
+# Genomic Address Service Clustering Workflow
 
-This is an example pipeline to be used for integration with IRIDA Next.
+This workflow takes provided JSON-formatted MLST profiles and converts them into a phylogenetic tree with associated flat cluster codes for use in [Irida Next](https://github.com/phac-nml/irida-next). The workflow also generates a interactive tree for visualization.
```

Comment:
Change "generates **a** interactive tree" to "generates **an** interactive tree"

Reply:
Updated in f01999f9c80d20c646211d7de65333dedb894c82